cmd/utils.go

```diff
@@ -103,7 +103,7 @@ func fileMustExist(filePath string) {
 	if err == nil {
 		f.Close()
 	} else {
-		fatalExitf(err, "Failed to create required file", filePath)
```

Comment:
why ignore the extra filePath info?

Reply:
It was my first try to use go and my IDE said that there are to much Parameters used in method fatalExitf, so I deleted it ;-)